src/app/qgisapp.cpp

```diff
@@ -1583,6 +1584,7 @@ void QgisApp::createActions()
   connect( mActionCircularStringCurvePoint, SIGNAL( triggered() ), this, SLOT( circularStringCurvePoint() ) );
   connect( mActionCircularStringRadius, SIGNAL( triggered() ), this, SLOT( circularStringRadius() ) );
   connect( mActionMoveFeature, SIGNAL( triggered() ), this, SLOT( moveFeature() ) );
+  connect( mActionMoveFeatureCopy, SIGNAL( triggered() ), this, SLOT( moveFeatureCopy() ) );
```

Comment:
https://github.com/qgis/QGIS-Enhancement-Proposals/issues/77  ;)

Actually this would be a good candidate for a lambda function to avoid the one line slot